azurerm_key_vault_access_policy/main.tf

```diff
@@ -0,0 +1,21 @@
+provider "azurerm" {
+  version = "=1.42.0"
+}
+
+terraform {
+  # The configuration for this backend will be filled in by Terragrunt
+  backend "azurerm" {}
+}
+
+resource "azurerm_key_vault_access_policy" "key_vault_access_policy" {
+  key_vault_id              = var.key_vault_id
+  tenant_id                 = var.tenant_id
+  object_id                 = var.object_id
+  application_id            = var.application_id
+
+  key_permissions           = var.key_permissions
+  secret_permissions        = var.secret_permissions
+  certificate_permissions   = var.certificate_permissions
```

Comment:
Leave only one space between  certificate_permissions and _=_ sign, and align all the other variables